deploy/build-configs/bc-ods-finish.yml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: ods-finish
+parameters:
+- name: GIT_URL
+  description: "Git repository URL"
+- name: SOURCE_SERCET
+  description: "Name of source secret to access GIT_URL"
+  value: cd-user-token
```

Comment:
@michaelsauter Not sure why we need this value. What if the repo we reference is publicly accessible and thus don't require to authenticate (as it is right now with ours)?